packages/orbit-components/src/Popover/components/ContentWrapper.tsx

```diff
@@ -198,6 +200,7 @@ const PopoverContentWrapper = ({
             shown ? "lm:opacity-100" : "lm:opacity-0",
           )}
           style={cssVars}
+          aria-label={ariaLabel}
```

Comment:
I feel that if we add ariaLabel we should also have the alternative aria-labelledby. Also, what is this aria-label supposed to say about the component? 🤔

Reply:
1. Sure, I can add it, it makes sense. Thx. 👍 
2. This `aria-label` attribute tells the menu's (element's) name inside the popover content component. If we have both of them, `aria-labelledby` is prioritized over `aria-label`. We can have it as a backup in case that aria-labelledby is not filled in. What do you think about this approach?



![image](https://github.com/user-attachments/assets/e2ed8534-8fc0-4c99-9b7a-552eb264d93d)


I've noticed now, that that's not `WCAG` violation, but just `Deque best practice`. I think it's not super critical to have it covered, but maybe we could include it as I already added it there (and it's optional attr).

https://dequeuniversity.com/rules/axe/4.10/aria-dialog-name?application=AxeChrome

Reply:
My question is: If we have a Popover with content inside, that content will be announced by the screen reader, right? If so, why do we need an aria-label? We will have the content announced anyway 🤔